graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/extensions/GraphQLSchemaExtensions.kt

```diff
@@ -34,46 +31,28 @@ import graphql.schema.idl.SchemaPrinter
  * @param includeDefaultSchemaDefinition boolean flag indicating whether SDL should include schema definition if using
  *   default root type names
  * @param includeDirectives boolean flag indicating whether SDL should include directive information
+ * @param includeDirectivesFilter Predicate to filter out specifc directives. Defaults to filter all directives by the value of [includeDirectives]
+ * @param descriptionsAsHashComments boolean flag indicating whether SDL print the description with # instead of ""
  */
 fun GraphQLSchema.print(
     includeIntrospectionTypes: Boolean = false,
     includeScalarTypes: Boolean = true,
     includeExtendedScalarTypes: Boolean = true,
     includeDefaultSchemaDefinition: Boolean = true,
-    includeDirectives: Boolean = true
+    includeDirectives: Boolean = true,
+    includeDirectivesFilter: Predicate<GraphQLDirective> = Predicate { includeDirectives },
+    descriptionsAsHashComments: Boolean = false
```

Comment:
should we even expose this since this is deprecated?

Reply:
Hmm, good point. That is less we would have to change later. I don't see the need for us to use it at Expedia Group. If someone else needs this to be backwards compatible later after 2.0 we can add it in until GraphQL Java removes it

Reply:
lets remove it then